src/logistration/tests/LoginPage.test.jsx

```diff
@@ -84,7 +84,7 @@ describe('LoginPage', () => {
       },
     });
     delete window.location;
-    window.location = { href: '' };
+    window.location = { href: '', search: '' };
```

Comment:
Why we are adding search in windows.location ?

Reply:
the test start failing because we added `getQueryParameters ` in `componentDidMount()`.
`getQueryParameters()` extracts query params from `global.location.search` and return those after parsing. Because this test is deleting the window.location, it was raising an error. 

I will see if we still need to set this field or not since we have moved from `getQueryParameters` to `searchParams`